From 5fb3c0c23ccd76c6f8239bbb66c3fb1230208d5e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luk=C3=A1=C5=A1=20Tyrychtr?= <ltyrycht@redhat.com>
Date: Mon, 11 Dec 2023 11:36:13 +0100
Subject: [PATCH] input-capture: Pass correct pointer type to
 gtk_window_add_group

It produced only a warning, but will be an error in the future.

Fixes #112
---
 src/inputcapture.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/inputcapture.c b/src/inputcapture.c
index ffa91a1d..bf60e6ea 100644
--- a/src/inputcapture.c
+++ b/src/inputcapture.c
@@ -409,7 +409,7 @@ create_input_capture_dialog (GDBusMethodInvocation *invocation,
   gtk_window_set_modal (GTK_WINDOW (dialog), TRUE);
 
   window_group = gtk_window_group_new ();
-  gtk_window_group_add_window (window_group, dialog);
+  gtk_window_group_add_window (window_group, GTK_WINDOW (dialog));
 
   dialog_handle = g_new0 (InputCaptureDialogHandle, 1);
   dialog_handle->request = g_object_ref (request);
-- 
GitLab

